### PR TITLE
fix: properly implement useMintedKolour

### PR DESCRIPTION
**Pull request by sk-umiuma**
resolves: https://app.asana.com/0/0/1204296603625125/f

### CODE REVIEW

File: pages/api/kolours/kolour/minted/detail.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  req: NextApiRequest,
+  res: NextApiResponse
+) {
+  try {
+    ClientError.assert(req.method === "GET", {
+      _debug: "invalid http method",
+    });
+    const { kolour } = req.query;
+    ClientError.assert(
+      typeof kolour === "string" && /^(?:[0-9A-F]{6})$/.test(kolour),
```
Comment (sk-kitsune) [inline pages/api/kolours/kolour/minted/detail.ts:19]: the regex can be simplified to `/^[0-9A-F]{6}$/`

File: modules/kolours/kolour.ts [hunk 2]
```diff
@@ -64,7 +64,8 @@ export async function getGenesisKreationWithKolours(
 }
 
 export async function getAllMintedKolours(
```
Comment (sk-shishi) [inline modules/kolours/kolour.ts:66]: Rename the function: getMintedKolours

Comment (sk-kitsune) [reply modules/kolours/kolour.ts:66]: `getAllMintedKolours` looks clearer to me. It indicates that the returned value is a list.

**Comment by sk-kitsune** [comment]
FE lgtm. @sk-shishi please help to review the BE part

**Review by sk-kitsune — APPROVED** [review]

File: modules/kolours/kolour.ts (hunk 2 above)
Comment (sk-shishi) [reply modules/kolours/kolour.ts]: No, because it doesn't return "all" in every case.

**Review by sk-kitsune — APPROVED** [review]
✅ **Approved**: no blocking issues

**Review by sk-kitsune — APPROVED** [review]
✅ **Approved**: no blocking issues

**Pull request merged**